is-the-average-whole/main.js

```diff
@@ -0,0 +1,17 @@
+function isTheAverageWhole(numbersArray) {
+  var sum = 0;
+  var avg = 0;
+  for (var i = 0; i < numbersArray.length; i++) {
+    sum += numbersArray[i];
+  }
+  avg = (sum / numbersArray.length - 1)
```

Comment:
you don't want to do `length-1` here, since that is just how you get to the last index. the `length` is the actual count of items in the array, which you will want to find the average 👍